runtime/runtime-core/jvm/test/aws/smithy/kotlin/runtime/retries/impl/Utils.kt

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package aws.smithy.kotlin.runtime.retries.impl
+
+import com.charleskorn.kaml.Yaml
+import kotlinx.serialization.DeserializationStrategy
+
+fun <T> Map<String, String>.deserialize(serializer: DeserializationStrategy<T>): Map<String, T> =
```

Comment:
**nit**: `deserializeYaml` would be more appropriate since this isn't namespaced in a way to tell us the expected format.